sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Platform/EnvironmentVariableConstants.cs

```diff
@@ -65,5 +65,10 @@ internal static class EnvironmentVariableConstants
         /// When set to true, exporter will emit resources as metric telemetry.
         /// </summary>
         public const string EXPORT_RESOURCE_METRIC = "OTEL_DOTNET_AZURE_MONITOR_ENABLE_RESOURCE_METRICS";
+
+        /// <summary>
+        /// When set to true, exporter will sample the logs based on configured sampling rate.
```

Comment:
This could be misleading, as logs sampling is not doing anything based on sampling rate. 

"When set to true, logs emitted within an active tracing context would follow the Activity's sampling decision" is more correct.

Reply:
Also "exporter will sample the logs" feels incorrect to read as "exporter" only exports, not samples :)